app/controllers/api/functions.php

```diff
@@ -1448,15 +1449,14 @@
     ->label('sdk.response.model', Response::MODEL_NONE)
     ->param('functionId', '', new UID(), 'Function ID.')
     ->param('deploymentId', '', new UID(), 'Deployment ID.')
-    ->param('buildId', '', new UID(), 'Build unique ID.')
+    ->param('buildId', '', new UID(), 'Build unique ID.', true) // added as optional param for backward compatibility
```

Comment:
What backwards compatibility are we worried about?

This is scoped for 1.6 which means we're going to be bumping the SDKs major version so breaking changes for that should be fine.

Reply:
The endpoint should continue to work for people using older SDKs as well